unit_tests/test_microbenchmarking.py

```diff
@@ -65,10 +65,12 @@ def _get_hash(*args, **kwargs):
 
 class TestMBM(unittest.TestCase):  # pylint: disable=too-many-public-methods
     def setUp(self):
-        self.mbra = MicroBenchmarkingResultsAnalyzerMock(email_recipients=('alex.bykov@scylladb.com', ))
+        self.mbra = MicroBenchmarkingResultsAnalyzerMock(email_recipients=(
+            'alex.bykov@scylladb.com', ), es_index="microbenchmarking")
         self.mbra.hostname = 'godzilla.cloudius-systems.com'
         self.cwd = os.path.join(os.path.dirname(__file__), '..', 'sdcm')
         self.mbra.test_run_date = "2019-06-27_11:39:40"
+        self.es_index = "microbenchmarking"
```

Comment:
don't we want some coverage that the new index also works ?

Reply:
Once we have more results in it and old microbenchmark will be removed, we switch to new index also unittests. 
Logic was not changed at all how microbenchmark get data, only index name

Reply:
I would recommend open a task for it, not forgetting that unittest should be part of removal of the old index